kubernetes/samples/scripts/create-weblogic-domain/model-in-image/sample-archive/wlsdeploy/applications/myapp/sample_war/index.jsp

```diff
@@ -0,0 +1,44 @@
+<%-- Copyright (c) 2019, 2020, Oracle Corporation and/or its affiliates. --%>
+<%-- Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl. --%>
+<%@ page import="javax.naming.InitialContext" %>
+<%@ page import="javax.management.*" %>
+<%@ page import="java.io.*" %>
+<%
+  InitialContext ic = null;
+  try {
+    String srName=System.getProperty("weblogic.Name");
+
+    out.println("<html><body><pre>");
+    out.println("**************************************************************");
+    out.println();
+    out.println("Hello World! Welcome to WebLogic server '" + srName + "'!");
+
+    ic = new InitialContext();
+
+    MBeanServer mbs = (MBeanServer)ic.lookup("java:comp/env/jmx/runtime");
+
+    ObjectName jvmRuntime  = new ObjectName("com.bea:ServerRuntime=" + srName + ",Name=" + srName + ",Type=JVMRuntime");
+    ObjectName jdbcRuntime = new ObjectName("com.bea:ServerRuntime=" + srName + ",Name=" + srName + ",Type=JDBCServiceRuntime");
+    ObjectName[] dataSources = (ObjectName[])mbs.getAttribute(jdbcRuntime, "JDBCDataSourceRuntimeMBeans");
+
+    out.println();
+    out.println("Free JVM Heap " + mbs.getAttribute(jvmRuntime, "HeapFreePercent") + "%.");
+    out.println();
+    out.println("Found " + dataSources.length + " data source" + (String)((dataSources.length!=1)?"s:":":"));
+
+    for (ObjectName dataSource : dataSources) {
+       String dsName  = (String)mbs.getAttribute(dataSource, "Name");
+       String dsState = (String)mbs.getAttribute(dataSource, "State");
+       out.println("  Datasource '" + dsName + "': State='" + dsState +"'");
+    }
+
+    out.println();
+    out.println("**************************************************************");
+
+  } catch (Throwable t) {
+    t.printStackTrace(new PrintStream(response.getOutputStream()));
```

Comment:
This won't show in the browser but goes to the stdout of the server

Reply:
I think it does show in the browser - I've tested it.   Note that it's not going to 'stdout' but to 'response.getOutputStream()'.